packages/components/src/components/hds/breadcrumb/item.ts

```diff
@@ -27,6 +33,23 @@ export interface HdsBreadcrumbItemSignature {
 }
 
 export default class HdsBreadcrumbItem extends Component<HdsBreadcrumbItemSignature> {
+  @tracked linkToExternal: LinkTo | null = null;
+
+  constructor(owner: Owner, args: HdsBreadcrumbItemSignature['Args']) {
+    super(owner, args);
+
+    // if want to make sure that we do not try to resolve the component if it's not needed
+    if (args.isRouteExternal) {
```

Comment:
@didoo I add this safeguard to we do not run it if do not need to however as you can see I did not modify unility function as I still want it to be safe contained, even though we might delete it in the future all together once ember-engines aren't a thing

Reply:
> I did not modify unility function as I still want it to be safe contained,
makes sense